recipes/implicit/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "implicit" %}
+{% set version = "0.2.7" %}
+{% set sha256 = "b8c5fd270a3335988d8cff33f6e31b721d6d13813a02e6f61e21de1d62526267" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not x86_64]
```

Comment:
Does it not compile 32-bit? Don't see that documented in the library, so just curious.